beacon_node/beacon_chain/src/state_advance_timer.rs

```diff
@@ -362,20 +341,18 @@ fn advance_head<T: BeaconChainTypes>(
                 .process_validator_statuses(state.current_epoch(), &summary, &beacon_chain.spec)
             {
                 error!(
-                    log,
-                    "Unable to process validator statuses";
-                    "error" => ?e
+                    error = ?e,
+                    "Unable to process validator statuses"
                 );
             }
         }
     }
 
     debug!(
-        log,
-        "Advanced head state one slot";
-        "head_block_root" => ?head_block_root,
-        "state_slot" => state.slot(),
-        "current_slot" => current_slot,
+        ?head_block_root,
+        state_slot = ?state.slot(),
```

Comment:
I think we want to keep this as `Display` formatting also
```suggestion
        state_slot = %state.slot(),
```